index.js

```diff
@@ -1,17 +1,78 @@
 class SortedList {
-  constructor() {}
 
-  add(item) {}
+  //Iteration 1: constructor()
+  constructor(items, length) {
+       this.items = [];
+      this.length = 0;     
+  }
 
-  get(pos) {}
+  // Iteration 2: add(item)
+  add(item) {
+    if (typeof item != 'undefined') {
+      // if(this.length === 0){
+      //   this.items.push(item);
+      // }else{
 
-  max() {}
+      // }
+      this.items.push(item);
+      this.items.sort((a, b) => (a - b));
+      this.length++;
+    }
+  }
 
-  min() {}
+  // Iteration 3: get(pos)
+  get(pos) {
+    if (this.items.length === 0 || pos >= this.items.length) {
+      throw new Error("OutOfBounds");
+    } else {
+      const item = this.items.slice(pos, (pos + 1));
+      if (typeof item === 'undefined') {
+        throw new Error("OutOfBounds");
+      } else {
+        return item;
+      }
+    }
+  }
 
-  sum() {}
+  //Iteration 4: max()
+  max() {
+    if (this.items.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return this.items.sort((a, b) => (a - b)).slice(this.length - 1);
```

Comment:
you shouldnt have to sort it, becasuse every time you add something, you immediately are sorting them, so no need to re-do that method